cloud-init.yml

```diff
@@ -0,0 +1,9 @@
+# cloud-config
+users:
+  - default
+  - name: chris
+    groups: sudo
+    sudo: ALL=(ALL) NOPASSWD:ALL
+    shell: /bin/bash
+    ssh_authorized_keys:
+      - # ADD OUR KEY HERE
```

Comment:
I tried adding my key and running it, but it errored out the first 7 lines. said it did not recognize the commands.